### DISCUSSION TITLE
What's the best way to feed x64dbg some debug info?

### DISCUSSION THREAD

**Discussion by u07** [x64dbg/x64dbg#3374] [General]
I have patched a bunch of dlls with another tool and remembered the addresses. How can I automatically pass that info to x64dbg, so that some comments appear near the patched instructions? 

I was thinking about saving that info as JSON database. But it requires a new database for every dll. There's no way to save multiple dll info into a single database, and no way to load it automatically when dll is loaded. 

The next idea was to create .map files, but it requires a plugin and still we have no auto-load here...

**Comment by mrexodia** [comment]
If you load an executable the database for that executable will contain comments for all DLLs and they will automatically be loaded. Most likely the easiest way is to generate a script and quickly execute it:

```
commentset my.dll:$rva, "<my comment>"
...
```

After you executed this script the information will be in the database for the main executable and you're good to go.

**Reply by u07** [reply]
Cool, that does the job! Thank you!
